### ISSUE TITLE
Unclosed client session output

### ISSUE THREAD

**Issue by AllPurposeBen** [bdr99/pymazda#5]
When running what I think is an valid variation of the example code, in addition to the desired json output, I see:
```
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x10b8ce7c0>
Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x10d337220>, 3.069700875)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x10d31d220>, 5.118501917)]']
connector: <aiohttp.connector.TCPConnector object at 0x10b8ce7f0>
```

Here's my test script:
```
import pymazda
import asyncio
import json

email = "email@domain.tld"
password = "thepassword"

async def test() -> None:
	# Initialize API client (MNAO = North America)
	client = pymazda.Client(email, password, "MNAO")

	# Get list of vehicles from the API (returns a list)
	vehicles = await client.get_vehicles()

	# Loop through the registered vehicles
	for vehicle in vehicles:
		# Get vehicle ID (you will need this in order to perform any other actions with the vehicle)
		vehicle_id = vehicle["id"]

		# Get and output vehicle status
		status = await client.get_vehicle_status(vehicle_id)
		#print(status)
		print(json.dumps(status, indent=2))

if __name__ == "__main__":
	loop = asyncio.get_event_loop()
	loop.run_until_complete(test())
```

What am I missing in closing the session properly? Apologies if it's obvious, I'm not great with python.

**Comment by bdr99** [comment]
Thanks for pointing this out! So what's happening is that internally, the library instantiates an instance of `aiohttp.ClientSession` to use for the API requests. This session needs to be closed when you're finished using it, but the library didn't expose a function for doing so. I just published version 0.0.10 which adds a `client.close()` function to close the session. The example code is updated to include this. Can you give it a try with the latest version, and let me know if it resolves your issue?

**Comment by AllPurposeBen** [comment]
Worked like a charm, thank you!
